ts/input/tex/html/HtmlMethods.ts

```diff
@@ -15,35 +15,35 @@
  *  limitations under the License.
  */
 
-
 /**
  * @fileoverview Methods for the Html package.
  *
  * @author v.sorge@mathjax.org (Volker Sorge)
  */
 
-
 import TexParser from '../TexParser.js';
-import {ParseMethod} from '../Types.js';
+import { ParseMethod } from '../Types.js';
 import NodeUtil from '../NodeUtil.js';
-import {ParseUtil} from "../ParseUtil.js";
-import {MmlNode} from '../../../core/MmlTree/MmlNode.js';
+import { ParseUtil } from '../ParseUtil.js';
+import { MmlNode } from '../../../core/MmlTree/MmlNode.js';
 import TexError from '../TexError.js';
 
 /** Regexp for matching non-characters as specified by {@link https://infra.spec.whatwg.org/#noncharacter}. */
-const nonCharacterRegexp = /[\u{FDD0}-\u{FDEF}\u{FFFE}\u{FFFF}\u{1FFFE}\u{1FFFF}\u{2FFFE}\u{2FFFF}\u{3FFFE}\u{3FFFF}\u{4FFFE}\u{4FFFF}\u{5FFFE}\u{5FFFF}\u{6FFFE}\u{6FFFF}\u{7FFFE}\u{7FFFF}\u{8FFFE}\u{8FFFF}\u{9FFFE}\u{9FFFF}\u{AFFFE}\u{AFFFF}\u{BFFFE}\u{BFFFF}\u{CFFFE}\u{CFFFF}\u{DFFFE}\u{DFFFF}\u{EFFFE}\u{EFFFF}\u{FFFFE}\u{FFFFF}\u{10FFFE}\u{10FFFF}]/u;
+const nonCharacterRegexp =
+  /[\u{FDD0}-\u{FDEF}\u{FFFE}\u{FFFF}\u{1FFFE}\u{1FFFF}\u{2FFFE}\u{2FFFF}\u{3FFFE}\u{3FFFF}\u{4FFFE}\u{4FFFF}\u{5FFFE}\u{5FFFF}\u{6FFFE}\u{6FFFF}\u{7FFFE}\u{7FFFF}\u{8FFFE}\u{8FFFF}\u{9FFFE}\u{9FFFF}\u{AFFFE}\u{AFFFF}\u{BFFFE}\u{BFFFF}\u{CFFFE}\u{CFFFF}\u{DFFFE}\u{DFFFF}\u{EFFFE}\u{EFFFF}\u{FFFFE}\u{FFFFF}\u{10FFFE}\u{10FFFF}]/u;
```

Comment:
This doesn't seem all that helpful.

Reply:
I know. But I feel we already have enough `prettier-ignore`...